### PR TITLE
Write a routing acceptance test

### PR DESCRIPTION
**Pull request by Gaurav0**
This adds an acceptance test using ember-cli-mirage and a whole bunch of interesting hacks.

### CODE REVIEW

**Comment by Gaurav0** [comment]
I got it working!!! Please review.

File: app/adapters/application.js [hunk 1]
```diff
@@ -1,8 +1,14 @@
 import DS from 'ember-data';
+import Ember from 'ember';
 import config from '../config/environment';
 
 export default DS.RESTAdapter.extend({
-  host:'https://api.github.com',
```
Comment (rwjblue) [inline app/adapters/application.js]: Can we store this in `config/environment.js`?  That allows us to have a different value in the test environment from development/production, and should allow you to remove the `host` computed.

**Comment by Gaurav0** [comment]
@rwjblue Updated.

File: tests/acceptance/routing-test.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    Ember.run(application, 'destroy');
+  }
+});
+
+test('Able to do routing in a gist', function(assert) {
+  const login = "Gaurav0";
+  const gist_id = "35de43cb81fc35ddffb2";
+  const commit = "f354c6698b02fe3243656c8dc5aa0303cc7ae81c";
+
+  function create_file(filename, content) {
```
Comment (joostdevries) [inline tests/acceptance/routing-test.js]: I'd move this to helpers

Comment (Gaurav0) [inline tests/acceptance/routing-test.js]: The main reason I haven't done so is that this uses closure over login, gist_id, and commit.

Comment (joostdevries) [inline tests/acceptance/routing-test.js]: Understood.

**Pull request merged**